docs_app/content/navigation.json

```diff
@@ -5,12 +5,12 @@
       "title": "Overview"
     },
     {
-      "url": "api",
-      "title": "Reference"
+    "url": "api",
+    "title": "Reference"
     },
     {
-      "url": "guide/v6/migration",
```

Comment:
reverted the change and added the new URL.
I guess as long as the URL is still reachable the menu can be changed? 
Otherwise, I'm not sure how to continue with the menu item.

https://github.com/ReactiveX/rxjs/pull/5128/commits/0e893c32af414b71d3e01b5bd97c5703162469a1